src/main/java/cz/sparko/boxitory/service/HashService.java

```diff
@@ -0,0 +1,8 @@
+package cz.sparko.boxitory.service;
+
+import java.io.File;
+
+public interface HashService {
+    String getHashType();
+    String getChecksum(File file);
```

Comment:
Argument should not be `File`. For example when we save hashes to database and get them from there. 
So maybe make parameter as generic type and implementation `DigestHashService` should type it as `File` and should be renamed to `DigestFileHashService`? Or something like that would ensure better extensibility than this.